examples/vite/src/index.scss

```
@@ -13,6 +13,7 @@ layer(stream-app-overrides);
 @import url('./AccessibilityNavigation/ReturnToSkipNavigation.scss')
 layer(stream-app-overrides);
 @import url('stream-chat-react/dist/css/emoji-picker.css') layer(stream-new-plugins);
+@import url('stream-chat-react/dist/css/channel-detail.css') layer(stream-new-plugins);
```

Stylelint (17.12.0)

[error] Expected "url('stream-chat-react/dist/css/channel-detail.css')" to be "'stream-chat-react/dist/css/channel-detail.css'" (import-notation)

(import-notation)